src/applications/yellow-ribbon/sass/yellow-ribbon.scss

```diff
@@ -0,0 +1,9 @@
+@media (min-width: 768px) {
+  .school-program {
+    margin-top: 36px !important;
+  }
+}
```

Comment:
Can the `medium-screen:vads-u-margin-top--4` not be used instead here? I'm not sure if the responsive prefixes support the margin utilities

Reply:
I actually played with that exact utility considerably but it was always off by 1-5px, so I decided instead to reach for custom styling here (the **ONLY** custom styling so far for YR! 😭 )